ButtonA: 1
ButtonB: 2
ButtonX: 0
ButtonY: 3
ButtonLB: 4
ButtonRB: 5
ButtonBack: 8
ButtonStart: 9
ButtonLeftThumb: 6
ButtonRightThumb: 7
ButtonLeftTrigger: 10
ButtonRightTrigger: 11
AxisLeftThumbX: 0
AxisLeftThumbY: 1
AxisRightThumbX: 2
AxisRightThumbY: 3
AxisDPadX: 4
AxisDPadY: 5
